app/models/conference/importer.rb

```diff
@@ -0,0 +1,73 @@
+require 'csv'
+class Conference::Importer
+  
+  ## This importer is depending on agreed-upon input format.
+  ## Input file:
+  # - Dates should be formatted with dd mm yyyy
+  # - UID should be unique and formatted with 'current season' + id: 2017001
+  # - The headers should not be changed
+  # Output:
+  # Conferences will be updated or created
+  # Conferences deleted in the input file, are not removed from the table
+  # Import errors are logged in a separate 'imports_logger' file
+  
+  class << self
+    def import(file)
```

Comment:
(Disclaimer: Nitpicking / Tangent :wink: )

I always find mirroring the class name in its only public method a bit odd: `Runner.run`. _of course_ it runs, hence the name of the class :upside_down_face: 

My personal approach is mostly to name the method `call`. `run` would work, too, but `call` is available on Lambdas/Procs so the architecture - theoretically - also gets more losely coupled.